Unity/opentok/holo/holo/Capturer.mm

```diff
@@ -0,0 +1,373 @@
+        }
+        NSData *data = nil;
+        if ([videoFrame isAugmented]) {
+            data = [NSData dataWithBytes:[videoFrame augmentingData] length:[videoFrame augmentingDataSize]];
+//            NSLog(@"[holo]: Capturer %p capturer augmented data size is %lu", self, static_cast<size_t>(data.length));
+        }
+        // TODO: Review the way we build the timestamp we provide through CMTimeMake (not sure is correct).
+        [_videoCaptureConsumer consumeImageBuffer:buffer
+                                      orientation:orientation
+                                        timestamp: CMTimeMake([videoFrame timeStampNs], CMTimeScale(NSEC_PER_SEC))
+                                         metadata:data];
+
+    }
+}
+
+- (void)setupListenerBlocks {
+}
+
+- (void)showCapturerError:(OTError*)error {
+    dispatch_async(dispatch_get_main_queue(), ^{
+        UIAlertController *alertController = [UIAlertController alertControllerWithTitle:@"[holo] Capturer"
+                                                                                 message:[NSString stringWithFormat:
+                                                                                          @"Capturer failed with error : %@", error.description]
+                                                                          preferredStyle:UIAlertControllerStyleAlert];
+        UIAlertAction *actionOk = [UIAlertAction actionWithTitle:@"Ok"
+                                                           style:UIAlertActionStyleDefault
+                                                         handler:nil];
+        [alertController addAction:actionOk];
+        [[[UIApplication sharedApplication] delegate].window.rootViewController
+         presentViewController:alertController
+         animated:YES completion:nil];
+    });
+}
+
+- (NSString*)captureSessionPreset {
+    return _capturePreset;
+}
+
+- (void)setCaptureSessionPreset:(NSString*)preset {
+}
+
+- (NSArray *)availableCaptureSessionPresets {
+    NSArray *allSessionPresets = [NSArray arrayWithObjects:
+                                  AVCaptureSessionPreset352x288,
+                                  AVCaptureSessionPreset640x480,
+                                  AVCaptureSessionPreset1280x720,
+                                  AVCaptureSessionPreset1920x1080,
+                                  AVCaptureSessionPresetPhoto,
+                                  AVCaptureSessionPresetHigh,
+                                  AVCaptureSessionPresetMedium,
+                                  AVCaptureSessionPresetLow,
+                                  nil];
+    return allSessionPresets;
+}
+
+- (double)activeFrameRate {
+    return 30.0;
+}
+
+- (BOOL)isAvailableActiveFrameRate:(double)frameRate {
+    return NO;
+}
+
+- (AVCaptureDevicePosition)cameraPosition {
+    return AVCaptureDevicePositionFront;
+}
+
+- (void)setCameraPosition:(AVCaptureDevicePosition) position {
+}
+
+- (NSArray*)availableCameraPositions {
```

Comment:
i dont think this function is used.

Reply:
It might be called by the OpenTok video client SDK.

Reply:
Removed it.